feat(curriculum): add quicksort lab to frontend cert (#56009)
freeCodeCamp/freeCodeCamp: client/src/pages/learn/front-end-development/lab-quick-sort/index.md

---

> title: Introduction to the Build a Quick Sort Algorithm

Reviewer: ```suggestion
title: Introduction to the Build the Quick Sort Algorithm
```

> title: Introduction to the Build the Quick Sort Algorithm

Reviewer 2: ```suggestion
title: Introduction to the Build the Quicksort Algorithm
```

> block: lab-quick-sort-algorithm

Reviewer 2: ```suggestion
block: lab-quicksort-algorithm
```

superBlock: front-end-development
---

## Introduction to the Build a Quick Sort Algorithm

> This is a test for the new project-based curriculum.

Reviewer: ```suggestion
For this lab, you will implement the Quick Sort algorithm using JavaScript.
```

> For this lab, you will implement the Quick Sort algorithm using JavaScript.

Reviewer 2: ```suggestion
## Introduction to the Build the Quicksort Algorithm

For this lab, you will implement the Quicksort algorithm using JavaScript.
```
